CHANGELOG.md

```diff
@@ -35,8 +35,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
     9. `contact`
     10. `contributors`
 - Changed the regular expression for PMC ID to allow 8 digits in addition to the preexisting 7. Issue [#443](https://github.com/citation-file-format/citation-file-format/issues/443); PR [#469](https://github.com/citation-file-format/citation-file-format/pull/469)
-- Instances of `person` now explicitly require one of `alias`, `email`, `given-names`, `family-names` or `orcid` to avoid accidentally having empty person objects (notably, `authors`). PR [#462](https://github.com/citation-file-format/citation-file-format/pull/462)
+- Instances of `person` now explicitly require one of `alias`, `email`, `given-names`, `family-names` or `orcid` to avoid accidentally having empty person objects (notably, `authors`); PR [#462](https://github.com/citation-file-format/citation-file-format/pull/462)
 - Fixed the regular expression for ISBN (should only affect ISBN10 strings that didn't have any dashes or spaces, whose check digit is `X`). Issue [#323](https://github.com/citation-file-format/citation-file-format/issues/323); PR [#335](https://github.com/citation-file-format/citation-file-format/pull/335); PR [#337](https://github.com/citation-file-format/citation-file-format/pull/337)
+- Loosen requirement of authors in references to allow for any of authors and/or editors. Issue [#334](https://github.com/citation-file-format/citation-file-format/issues/334); PR [#524](https://github.com/citation-file-format/citation-file-format/pull/524)
```

Comment:
Not sure this qualifies as a fix. From a semver perspective it's more of an addition, right?

Reply:
Depending on whether you see the inability to represent edited volumes correctly in `references` as a bug. I think this is a reasonable interpretation, and thus think the change should be in "# Fixes".

Reply:
Decided to move this into **Added** as per semver.

Reply:
Done in https://github.com/citation-file-format/citation-file-format/pull/524/commits/6de9a66402846bf9459710f5df8edb3f30feaf0f.